shell/platform/common/accessibility_bridge.cc

```diff
@@ -649,8 +656,63 @@ gfx::NativeViewAccessible AccessibilityBridge::GetNativeAccessibleFromId(
 gfx::RectF AccessibilityBridge::RelativeToGlobalBounds(const ui::AXNode* node,
                                                        bool& offscreen,
                                                        bool clip_bounds) {
-  return tree_.RelativeToTreeBounds(node, gfx::RectF(), &offscreen,
-                                    clip_bounds);
+  return tree_->RelativeToTreeBounds(node, gfx::RectF(), &offscreen,
+                                     clip_bounds);
+}
+
+ui::AXNode* AccessibilityBridge::GetNodeFromTree(
+    ui::AXTreeID tree_id,
+    ui::AXNode::AXID node_id) const {
+  return GetNodeFromTree(node_id);
+}
+
+ui::AXNode* AccessibilityBridge::GetNodeFromTree(
+    ui::AXNode::AXID node_id) const {
+  return tree_->GetFromId(node_id);
+}
+
+ui::AXTreeID AccessibilityBridge::GetTreeID() const {
+  return tree_->GetAXTreeID();
+}
+
+ui::AXTreeID AccessibilityBridge::GetParentTreeID() const {
+  return ui::AXTreeIDUnknown();
+}
+
+ui::AXNode* AccessibilityBridge::GetRootAsAXNode() const {
+  return tree_->root();
+}
+
+ui::AXNode* AccessibilityBridge::GetParentNodeFromParentTreeAsAXNode() const {
+  return nullptr;
+}
+
+ui::AXTree* AccessibilityBridge::GetTree() const {
+  return tree_.get();
+}
+
+ui::AXPlatformNode* AccessibilityBridge::GetPlatformNodeFromTree(
+    const ui::AXNode::AXID node_id) const {
+  auto platform_delegate_weak = GetFlutterPlatformNodeDelegateFromID(node_id);
+  if (platform_delegate_weak.expired()) {
+    return nullptr;
```

Comment:
Is this case unnecessary due to `lock` check below? See: https://en.cppreference.com/w/cpp/memory/weak_ptr/lock

Reply:
Probably